server/message_hooks.go

```diff
@@ -119,26 +119,24 @@ func (p *Plugin) MessageHasBeenPosted(_ *plugin.Context, post *model.Post) {
 			return
 		}
 
-		isSelfPost := len(members) == 1
-		chatMembersSpanPlatforms := false
-		if !isSelfPost {
-			chatMembersSpanPlatforms, err = p.ChatMembersSpanPlatforms(members)
-			if err != nil {
-				p.API.LogWarn("Failed to check if chat members span platforms", "error", err.Error(), "post_id", post.Id, "channel_id", post.ChannelId)
-				return
-			}
-
-			if p.getConfiguration().SelectiveSync && !chatMembersSpanPlatforms {
-				return
+		chatShouldSync := false
+		if p.getConfiguration().SelectiveSync {
+			isSelfPost := len(members) == 1
+			if !isSelfPost {
```

Comment:
I'm wondering if/why we need to distinguish self chats anymore? Can you elaborate?

Reply:
removed...not sure when/if self chats should sync.